racetime/templates/racetime/race/aside.html

```diff
@@ -24,6 +24,12 @@
             Reconnecting to chat…
         </span>
     </div>
+    <div class="scrollwarning" title="Click here to scroll to the bottom" onwheel="UpdateScroll()">
```

Comment:
1. Title text is unnecessary I think
1. `onwheel="UpdateScroll()"` can also be removed (don't see that function defined anywhere)

Reply:
Previous attempt that forgot to be cleaned up.

Reply:
Also in regards to title, I've kept it as you have it for disconnected as well. I'd lean to getting rid of both if we look to get rid of this one to stay consistent.